src/hooks/words/use-put-word-cache.hook.ts

```diff
@@ -0,0 +1,106 @@
+import { putWordByIdApi } from '@/api/words/put-word-by-id.api'
+import {
+  WordDataModifiable,
+  WordDataModifiableKey,
+} from '@/api/words/words.interface'
+import { isEmptyObjectHandler } from '@/handlers/is-empty-object.handler'
+import { modifyingWordFamily, wordsFamily } from '@/recoil/words.state'
+import { useCallback } from 'react'
```

Comment:
Generally speaking the use put word cache is too big.
And therefore there should be use put word cache by key  (Should return resetter too, prolly.)
and use put word cache kind of depend on the put word cache by key for resetting values?

Reply:
Also feel like this use put word cache is too long.

Reply:
refactored.